docs/source/setup.rst

```diff
@@ -4,9 +4,38 @@ Setup
 This page describes the procedure to set up API keys to authenticate your login and
 various ways to use the generated API key to log in to the client.
 
+Authentication with JupyterHub
+******************************
+
+This section is ONLY applicable for users running Modelon Impact in a JupyterHub environment.

```

Comment:
Maybe we could add something like:
, otherwise skip to :ref:\`below\<Authentication with API keys\>\`.